README.md

```diff
@@ -158,6 +158,7 @@ Ensure you have installed correct versions of necessary dependencies:
   [build.assets/Makefile](https://github.com/gravitational/teleport/blob/master/build.assets/Makefile#L21)
   (search for `RUST_VERSION`)
 * For `tsh` version > `10.x` with FIDO support, you will need `libfido` and `openssl 1.1` installed locally
+* To build the web UI, [`yarn`](https://yarnpkg.com/getting-started/install) is required.
```

Comment:
Could you also add a note about the `docker-ui` target. 

If the UI is first built with the make target `docker-ui` then no dependencies are required on the host machine other than `docker`.

Reply:
That's a nice alternative to have. Not everyone might want to install Node.js 😄 

I'll add it.